src/commands/template.rs

```diff
@@ -99,10 +100,26 @@ pub async fn template(args: Template, global: GlobalArgs) -> Result<()> {
 
     pb.set_message("Downloading competition template...");
     match download_archive(download_url, &destination, &pb).await {
-        Ok(_) => pb.finish_with_message(format!(
-            "Competition template downloaded to {}",
-            destination.display()
-        )),
+        Ok(_) => {
+            init_repository(&pb, global.project.as_path(), None).map_err(|err| {
```

Comment:
why not stick `format_permission_error` in `crate::error` and reuse it here? also init_repository  could also just return `create::Result<()>` to save yourself some unnecessary error formatting